tests/s2n_test.h

```diff
@@ -79,7 +79,8 @@
 
 #define FAIL()      FAIL_MSG("")
 
-#define FAIL_MSG( msg ) do { if (isatty(fileno(stdout))) { \
+#define FAIL_MSG( msg ) do { s2n_print_stacktrace(stdout); \
```

Comment:
the print stacktrace should be checking if `s_s2n_stack_traces_enabled`?

Reply:
Fixed